etcdserver/api/membership/cluster.go

```diff
@@ -624,6 +624,36 @@ func (c *RaftCluster) IsReadyToRemoveMember(id uint64) bool {
 	return true
 }
 
+func (c *RaftCluster) IsReadyToPromoteMember(id uint64) bool {
+	nmembers := 1
+	nstarted := 0
+
+	for _, member := range c.members {
```

Comment:
Rebase and use `c.VotingMembers()` instead. Ref: #24.

Reply:
will do